mmseg/models/backbones/mscan.py

```diff
@@ -0,0 +1,526 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Originally from https://github.com/visual-attention-network/segnext
+import math
+import warnings
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import build_activation_layer, build_norm_layer
+from mmcv.cnn.bricks import DropPath
+from mmcv.cnn.utils.weight_init import (constant_init, normal_init,
+                                        trunc_normal_init)
+from mmcv.runner import BaseModule
+
+from mmseg.models.builder import BACKBONES
+
+
+class DWConv(BaseModule):
+    """Depthwise Convolution Module.
+
+    This module is not removed because original pretrained models are used for
+    training, which has pretrained weights of related parameters.
+
+    Args:
+    channels (int): The dimension of features. Defaults: 768.
+    """
+
+    def __init__(self, channels=768):
+        super(DWConv, self).__init__()
+        self.dwconv = nn.Conv2d(
+            channels, channels, 3, 1, 1, bias=True, groups=channels)
+
+    def forward(self, x):
+        x = self.dwconv(x)
+        return x
+
+
+class Mlp(BaseModule):
+    """Multi Layer Perceptron (MLP) Module.
+
+    Args:
+    in_features (int): The dimension of input features.
+    hidden_features (int): The dimension of hidden features.
+        Defaults: None.
+    out_features (int): The dimension of output features.
+        Defaults: None.
+    act_cfg (dict | None): Config dict for activation layer in block.
+        Default: dict(type='GELU').
+    drop (float): The number of dropout rate in MLP block.
+        Defaults: 0.0.
+    """
```

Comment:
here is always indentation
```suggestion
    Args:
        in_features (int): The dimension of input features.
        hidden_features (int): The dimension of hidden features.
            Defaults: None.
        out_features (int): The dimension of output features.
            Defaults: None.
        act_cfg (dict | None): Config dict for activation layer in block.
            Default: dict(type='GELU').
        drop (float): The number of dropout rate in MLP block.
            Defaults: 0.0.
    """
```